content/dart/concepts/list/terms/clear/clear.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: 'Clear Method (List)'
+Description: 'Removes all objects from the list; the length of the list becomes zero.'
+Subjects:
+  - 'Computer Science'
+  - 'Mobile Development'
+  - 'Code Foundations'
+Tags:
+  - 'Dart'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+The **`clear()`** method in Dart is used to remove all elements from a List, leaving an empty list in its place. This method is handy when you need to reset or clear the contents of a List, making it ready for new data.
+
+## Implementation
+
+```pseudo
+void clear();
+```
+
+## Parameters
+
+This method does not take any parameters.
+
+## Return Type
+
+This method does not return any value.
```

Comment:
This section is not required too but you can add this line in the introductory paragraph.